websites/F/FreeCodeCamp/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "author": {
+      "name": "tom",
+      "id": "266024212978728963"
+    },
+    "service": "FreeCodeCamp",
+    "description": {
+      "en": "Want to learn code? Well freecodecamp has 11 languages you can choose from"
+    },

```

Comment:
maybe use their official description?
`Learn to code. Build projects. Earn certifications. Since 2015, 40,000 graduates have gotten jobs at tech companies including Google, Apple, Amazon, and Microsoft.`